### PR TITLE
Bug 1754042: Retry 429 and 503 errors in registry client

### PR DESCRIPTION
**Pull request by smarterclayton**
Add general across the board retries on 429 and 503 in registry client
commands that share the context.

Depends on openshift/library-go#534

### CODE REVIEW

**Comment by smarterclayton** [comment]
@dmage bumped vendor, updated with retries.  PTAL

File: pkg/cli/image/manifest/manifest.go [hunk 1]
```diff
@@ -87,6 +87,7 @@ func (o *SecurityOptions) Context() (*registryclient.Context, error) {
 	}
 	context, err := o.NewContext()
 	o.CachedContext = context
+	o.CachedContext.Retries = 3
```
Comment (dmage) [inline pkg/cli/image/manifest/manifest.go]: Maybe it's better to do it as the last step in `o.NewContext()`?

Comment (smarterclayton) [reply pkg/cli/image/manifest/manifest.go]: `Context()` is "I want the defaults for this command" while `NewContext` is "read from options only".  So Context

**Pull request merged**